guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -66,13 +73,20 @@ To update the certificates on a currently running Katello installation, run:
     --certs-update-server --certs-update-server-ca
 ----
 endif::[]
-
++
+Note that you must not access or modify `/root/ssl-build`.
+ifdef::satellite[]
+Contact Red Hat Support for further assistance.
+endif::[]
```

Comment:
If this means assistance if individual accesses/modifies the said file, I would probably omit this since you specifically tell them not to do it. If they do then contacting support is imho implied and does not need to be mentioned explicitly.

Reply:
Right. I will convince to relevant team.